pkgs/development/python-modules/clikit/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi,
```

Comment:
Coding conventions say when this doesn't fit on one line it should be written as
```
{ lib , buildPythonPackages, fetchPypi
, isPy27, ...
```

> https://nixos.org/nixpkgs/manual/#sec-syntax